Sources/NIOCore/SingleStepByteToMessageDecoder.swift

```diff
@@ -230,7 +236,8 @@ public final class NIOSingleStepByteToMessageProcessor<Decoder: NIOSingleStepByt
         return result
     }
 
-    private func decodeLoop(decodeMode: DecodeMode, seenEOF: Bool = false, _ messageReceiver: (Decoder.InboundOut) throws -> Void) throws {
+    @inlinable
+    func decodeLoop(decodeMode: DecodeMode, seenEOF: Bool = false, _ messageReceiver: (Decoder.InboundOut) throws -> Void) throws {
```

Comment:
Can you prefix this with an underscore so we know it was supposed to be private?